.github/workflows/tests.yml

```diff
@@ -0,0 +1,58 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          target: wasm32-unknown-unknown
+          toolchain: nightly
+          override: true
+      - run: cargo b --all --release
+      - run: cargo t --all --release
```

Comment:
Same as [here](https://github.com/consensus-shipyard/ipc-atomic-execution/pull/1#discussion_r1012743502).

Reply:
I still think `dev` build profile suits here better because it has:
```
debug-assertions = true
overflow-checks = true
```
This, it can catch more bugs.